routing/turns_sound_settings.hpp

```diff
@@ -50,11 +90,18 @@ class Settings
   bool IsValid() const;
 
   /// \brief computes the distance an end user shall be informed about the future turn
-  /// before it, taking into account speedMetersPerSecond and fields of the structure.
+  /// before it, taking into account speedMetersPerSecond and fields of the class.
   /// \param speedMetersPerSecond is a speed. For example it could be a current speed of an end
   /// user.
-  /// \return distance in units which are set in m_lengthUnits. (Meters of feet for the time being.)
-  double ComputeTurnDistance(double speedUnitsPerSecond) const;
+  /// \return distance in meters.
+  uint32_t ComputeTurnDistanceM(double speedMetersPerSecond) const;
+
+  /// \brief computes the distance which is needed for pronouncing turn sound notification
```

Comment:
computes the distance which will be passed at the |speedMetersPerSecond| while pronouncing turn sound notification